scene/src/components/backpack/WearableCategoryButton.tsx

```diff
@@ -0,0 +1,125 @@
+import { UiEntity, type UiTransformProps } from '@dcl/sdk/react-ecs'
+import { type WearableCategory } from '../../service/wearable-categories'
+import ReactEcs, {
+  type ReactElement,
+  type UiBackgroundProps
+} from '@dcl/react-ecs'
+import { getCanvasScaleRatio } from '../../service/canvas-ratio'
+import { Color4 } from '@dcl/sdk/math'
+import { COLOR } from '../color-palette'
+import Icon from '../icon/Icon'
+import type { AtlasIcon, URNWithoutTokenId } from '../../utils/definitions'
+import { noop } from '../../utils/function-utils'
+import { getBackgroundFromAtlas } from '../../utils/ui-utils'
+import { catalystWearableMap } from '../../utils/wearables-promise-utils'
+import { ROUNDED_TEXTURE_BACKGROUND } from '../../utils/constants'
+
+type WearableCategoryButtonProps = {
+  uiTransform?: UiTransformProps
+  category: WearableCategory
+  active?: boolean
+  onClick?: () => void
+  selectedURN: URNWithoutTokenId | null
+}
+export function WearableCategoryButton({
+  category,
+  uiTransform,
+  active,
+  onClick,
+  selectedURN
+}: WearableCategoryButtonProps): ReactElement {
+  const callbacks: { onClick: () => void } = {
+    onClick: onClick ?? noop
+  }
+  const canvasScaleRatio = getCanvasScaleRatio()
+  const categoryIcon: AtlasIcon = {
+    spriteName: category,
+    atlasName: 'backpack'
+  }
+  const iconSize = 34 * canvasScaleRatio * 2
+  const thumbnailSize = iconSize * 1.7
+
+  const textureProps: UiBackgroundProps =
+    selectedURN === null
+      ? getBackgroundFromAtlas({
+          atlasName: 'backpack',
+          spriteName: 'nft-empty'
+        })
+      : {
+          texture: {
+            src: `https://peer.decentraland.org/lambdas/collections/contents/${selectedURN}/thumbnail`
+          },
+          textureMode: 'stretch'
+        }
```

Comment:
small change: i think inverting to avoid using `selectedURN === null`, is more redeable
but if you prefer this way, I don't see issues about it
```suggestion
const textureProps: UiBackgroundProps = selectedURN
  ? {
      texture: {
        src: `https://peer.decentraland.org/lambdas/collections/contents/${selectedURN}/thumbnail`
      },
      textureMode: 'stretch'
    }
  : getBackgroundFromAtlas({
      atlasName: 'backpack',
      spriteName: 'nft-empty'
    })
```

Reply:
ESLint: Unexpected nullable string value in conditional. Please handle the nullish/ empty cases explicitly.(@typescript-eslint/ strict-boolean-expressions)

https://github.com/dclexplorer/bevy-ui-scene/commit/cc99157f68ecea3fdcf00d3a447e6b75e15cc4e0